### PR TITLE
Add new plans via planning.txt

### PR DESCRIPTION
**Pull request by willtheorangeguy**
Creating the new branch and planning information to get ready to start _BETA v0.0.3_ development.

### CODE REVIEW

**Comment by willtheorangeguy** [comment]
DO NOT DELETE or MERGE THIS BRANCH. We need it to continue development.

File: PLANNING.txt [hunk 1]
```diff
@@ -1,14 +1,35 @@
...
 - Travis CI or Circle CI integration.
-- Options screen help.
 
+Future Projects and Ideas (Anyone is free to contribute here):
+--------------------------------------------------------------
+- Rather than and inventory / hot bar, maybe more like a deck of cards (Clash Royale ref.). Or when battling a more of Boom Beach interface in which there are cards that you select and attack with.
+- Takes time to upgrade walls and defence (2-6 hours, depending on upgrade).
+- Could train mobs to become defensive mobs that do your defensive work for you if you convert them with potion, food, etc.
+- Mobs:
+	- Goats
```
Comment (Derpyface) [inline PLANNING.txt]: -Zombie (Would be the first mob because of attacking heart)

Comment (willtheorangeguy) [inline PLANNING.txt]: What do you mean by attacking heart?

File: PLANNING.txt [hunk 2]
```diff
@@ -1,14 +1,37 @@
...
+Future Projects and Ideas (Anyone is free to contribute here):
+--------------------------------------------------------------
+- Rather than and inventory / hot bar, maybe more like a deck of cards (Clash Royale ref.). Or when battling a more of Boom Beach interface in which there are cards that you select and attack with.
+- Takes time to upgrade walls and defence (2-6 hours, depending on upgrade).
+- Could train mobs to become defensive mobs that do your defensive work for you if you convert them with potion, food, etc.
+- Mobs could try to attack you, but also would attack something called a 'heart' (kind of like a beacon) and if that was destroyed then you would die. But of course you could still die from being killed by mobs.
+- Mobs:
+	- Goats
+	- Bears
+	- Zombies
```
Comment (willtheorangeguy) [inline PLANNING.txt]: Added zombies as per request in pull #5 , also added the 'heart' instance as in #5

**Comment by willtheorangeguy** [comment]
Will merge pull soon as we are now moving BETAv0.0.3 development to the `master` branch.

**Pull request merged**